docs/example-workflows/aws-automation-example.md

```diff
@@ -0,0 +1,70 @@
+# Example workflow: Detecting events in AWS and importing to Snyk
+## Detecting new ECR images and/or Lambda functions in AWS and importing/testing them with Snyk
+Using AWS's services, specifically [EventBridge](https://console.aws.amazon.com/events/), you can now detect almost any processes from other services, including ECR and Lambda, and trigger a Lambda function to be invoked upon that detection.
+Here are some highlights on how to configure this (through the UI and taken from [this guide](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-run-lambda-schedule.html)):
+1. Create a [Lambda function](https://console.aws.amazon.com/lambda/) that will receive data from EventBridge (you can create an empty fuction and work on it later), the data varies according to the event, for example: a push image to ECR event will have thise format and data (taken from [this guide](https://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr-eventbridge.html#:~:text=The%20following%20event%20is%20sent%20when%20each%20image%20push%20is%20completed.%20For%20more%20information%2C%20see%20Pushing%20a%20Docker%20image.)):
+```
+{
+  "account": "123456789012",
+  "detail": {
+    "action-type": "PUSH",
+    "image-digest": "sha256:f98d67af8e53a536502bfc600de3266556b06ed635a32d60aa7a5fe6d7e609d7",
+    "image-tag": "latest",
+    "repository-name": "ubuntu",
+    "result": "SUCCESS"
+  },
+  "detail-type": "ECR Image Action",
+  "id": "4f5ec4d5-4de4-7aad-a046-56d5cfe1df0e",
+  "region": "us-east-1",
+  "resources": [],
+  "source": "aws.ecr",
+  "time": "2019-08-06T00:58:09Z",
+  "version": "0"
+}
+```
+2. Go to your [Events page in AWS](https://console.aws.amazon.com/events/)
+3. Create a rule, for example: detecting a successfully pushed image to ECR
+- The event pattern will look somthing like this:
```

Comment:
should this be a bullet point? Looks like an odd one out

Reply:
Done